carousel.html

```diff
@@ -35,10 +37,298 @@
   </header>
 
     <main>
+      <div class="carousel__wrapper">
```

Comment:
As we talked on the meeting, you could keep the first element of the feature as a `block` from BEM convention 👌

Reply:
Got it 👍